schemas/com.shopify/create-collection/jsonschema/1-0-0

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.shopify",
+    "name" : "create-collection",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "type" : "object",
+  "properties" : {
+    "body_html" : {
```

Comment:
I think we need to remove it.

Reply:
Is'nt that a valuable info? Why would you drop it?

Reply:
My reasoning here is that:

1. We need to cleanup this up from HTML-markup, which can be quite messy.
2. There's no way to JOIN or aggregate this data, especially without sophisticated tools such as Spark
3. It can be big (especially with markup), up to the point where we hit Kinesis/Redshift max record/column limits.

Though I'm totally happy to reconsider.